src/helpers/github.ts

```diff
@@ -103,6 +103,12 @@ export const createIssue = async (
     const apiUrl = `${GITHUB_API_URL}/repos/${organization}/${repository}/issues`;
 
     const timeCapitalized = capitalizeWords(timeEstimate);
+    let titleCapitalized = capitalizeWords(issueTitle);
+
+    // remove punctuation at the end of the titleCapitalized if it exists
+    if (titleCapitalized[titleCapitalized.length - 1].match(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g)) {
+      titleCapitalized = titleCapitalized.slice(0, -1);
```

Comment:
It punctuates 100% of the time?

Reply:
I don't know, that's why I checked if the end is a punctuation